classes/Cart.php

```diff
@@ -1255,7 +1255,7 @@ public function getProductQuantity($idProduct, $idProductAttribute = 0, $idCusto
         $parentSql = 'SELECT
             COALESCE(SUM(first_level_quantity) + SUM(pack_quantity), 0) as deep_quantity,
             COALESCE(SUM(first_level_quantity), 0) as quantity
-          FROM (' . $firstUnionSql . ' UNION ' . $secondUnionSql . ') as q';
+          FROM (' . $firstUnionSql . ($productIsPack ? ' UNION ' . $secondUnionSql : '') . ') as q';
```

Comment:
This looks weird :-)

Reply:
You're right but it's the simplest working solution. Do you have a suggestion?